src/garage/torch/policies/gaussian_mlp_policy.py

```diff
@@ -18,36 +17,54 @@ class GaussianMLPPolicy(GaussianMLPModule, Policy):
         module : GaussianMLPModule to make prediction based on a gaussian
         distribution.
     :return:
-
     """
 
     def __init__(self, env_spec, **kwargs):
-        self._env_spec = env_spec
         self._obs_dim = env_spec.observation_space.flat_dim
         self._action_dim = env_spec.action_space.flat_dim
 
+        Policy.__init__(self, env_spec)
         GaussianMLPModule.__init__(self,
                                    input_dim=self._obs_dim,
                                    output_dim=self._action_dim,
                                    **kwargs)
 
     def forward(self, inputs):
         """Forward method."""
-        return super().forward(inputs)
+        return super().forward(torch.Tensor(inputs))
 
     def get_action(self, observation):
         """Get a single action given an observation."""
         with torch.no_grad():
             observation = observation.unsqueeze(0)
             dist = self.forward(observation)
-            std = dist.variance**0.5
-            return dist.rsample().squeeze(0).numpy(), dict(
-                mean=dist.mean.squeeze(0), log_std=np.log(std).squeeze(0))
+            return (dist.rsample().squeeze(0).numpy(),
+                    dict(mean=dist.mean.squeeze(0).numpy(),
+                         log_std=(dist.variance**.5).log().squeeze(0).numpy()))
 
     def get_actions(self, observations):
         """Get actions given observations."""
         with torch.no_grad():
             dist = self.forward(observations)
-            std = dist.variance**0.5
-            return dist.rsample().detach().numpy(), dict(mean=dist.mean,
-                                                         log_std=np.log(std))
+            return (dist.rsample().numpy(),
+                    dict(mean=dist.mean.numpy(),
+                         log_std=(dist.variance**.5).log().numpy()))
+
+    def get_log_likelihood(self, observation, action):
```

Comment:
these functions are differentiable right? in that case, i think you can omit the `get` part of the name. we tend to use `get` to distinguish non-differentiable functions.

Reply:
[Pytorch description and sample code](https://pytorch.org/docs/stable/distributions.html) shows they are differentiable.
I will change the name.